.github/dependabot.yml

```diff
@@ -13,12 +13,10 @@ updates:
     schedule:
       interval: "monthly"
     groups:
-      dep-patch-updates:
-        update-types:
-          - "patch"
-      dep-minor-updates:
+      minor-and-patch:
```

Comment:
**commit message:**

I'd like to see some reasoning in the commit message, e.g. "Decrease the PR noise from dependabot by merging both 'patch' and 'minor' updates".